inspirehep/modules/relations/command_producers.py

```diff
@@ -0,0 +1,218 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+
+def produce_labels_string(labels):
+    return ''.join([':' + label for label in labels if label])
+
+
+def produce_properties_string(properties):
+    if properties:
+        properties_strings = []
+        for key, value in properties.items():
+            value = str(value)
+            properties_string = key + ':' + value
+            properties_strings.append(properties_string)
+
+            return '{{{}}}'.format(','.join(properties_strings))
+    else:
+        return ''
+
+def produce_element_block(labels=None, properties=None, variable_name=None):
+    if not labels: labels = []
+    if not properties: properties = {}
+    if not variable_name: variable_name = ''
```

Comment:
`labels` and `properties` might be fine since ` produce_element_block(labels=[],..)` is bad idea in Python. But for `variable_name` you could just add a default value of empty string (and avoid this `if`)